lib/elixir/lib/io.ex

```diff
@@ -252,13 +257,14 @@ defmodule IO do
   @doc """
   Inspects `item` according to the given options using the IO `device`.
 
-  See `Inspect.Opts` for a full list of options.
+  See `inspect/2` for a full list of options.
   """
   @spec inspect(device, item, Keyword.t) :: item when item: var
   def inspect(device, item, opts) when is_list(opts) do
+    tag    = if (tag = opts[:tag]), do: [to_string(tag), ": "], else: ""
```

Comment:
@michalmuskala `IO.puts/1`, not `IO.gets/1` :) That's because atoms implement `String.Chars`.